example/C/src/ROS/ROSSerialization.lf

```diff
@@ -0,0 +1,98 @@
+/**
+ * A simple source-sink example that uses the federated Lingua Franca runtime to
+ * send ROS messages of type std_msgs::msg::Int32. The serialization/deserialization
+ * techniques are exposed here in the sender/receiver reactions. This mechanism should
+ * work for any ROS2 message type.
+ * 
+ * Note: the terminal must be properly sourced for ROS2. See 
+ * https://docs.ros.org/en/foxy/Tutorials/Configuring-ROS2-Environment.html.
+ *
+ * There is a variant of this example called ROSBuiltInSerialization.lf that uses
+ * the built-in ROS 2 serialization feature that is provided in LF.
+ */
+target C {
+    coordination: decentralized,
+    timeout: 10 sec,
+    cmake-include: "include/CMakeListsExtension.txt",
+    files: ["include/ament_target_dependencies.cmake"],
+    cmake: true,
```

Comment:
No longer necessary, right?

```suggestion
```

Reply:
That is right. Thank you for pointing that out.